src/execution/execute.ts

```diff
@@ -170,6 +174,7 @@ export interface ExecutionContext {
   abortSignalListener: AbortSignalListener | undefined;
   completed: boolean;
   cancellableStreams: Set<CancellableStreamRecord> | undefined;
+  propagateErrors: boolean;
```

Comment:
Should we allow people to toggle this on, i.e. allow for a GraphQL server to say we don't allow for this at the moment as this is an experimental feature? My inclination is that if we PR this to v16 we should default to off and v17 we could just do it this way, kind of want to spark the conversation here.

Reply:
We added such a flag in the [graphql-java PR](https://github.com/graphql-java/graphql-java/pull/3772). I'm generally in favor. Not too much for potential clashes, I think those are very very low probability but more for signaling the experimental status of the directive.

Reply:
Now that the directive is[ named `@experimental_disableErrorPropagation`](https://github.com/graphql/graphql-js/pull/4348/commits/f96492c096c373f2287cffc3328258245a020b82), I'm tempted to say no need for an opt-in flag. The name of the directive should add enough friction that users realize they're doing something experimental.

Reply:
Users should still have to add it to their schema, we don’t want it printed in every printSchema call with no opt-out.